src/Compilers/CSharp/Portable/Symbols/Source/ModifierUtils.cs

```diff
@@ -62,6 +62,27 @@ internal static class ModifierUtils
             return result;
         }
 
+        internal static void ReportDefaultInterfaceImplementationModifiers(
+            LanguageVersion availableVersion,
+            LanguageVersion requiredVersion,
+            DeclarationModifiers modifiers,
+            DeclarationModifiers defaultInterfaceImplementationModifiers,
+            Location errorLocation,
+            DiagnosticBag diagnostics)
+        {
+            DeclarationModifiers errorModifiers = modifiers & defaultInterfaceImplementationModifiers;
+            while (errorModifiers != DeclarationModifiers.None)
+            {
+                DeclarationModifiers oneError = errorModifiers & ~(errorModifiers - 1);
+                Debug.Assert(oneError != DeclarationModifiers.None);
+                errorModifiers = errorModifiers & ~oneError;
+                diagnostics.Add(ErrorCode.ERR_DefaultInterfaceImplementationModifier, errorLocation,
+                                ConvertSingleModifierToSyntaxText(oneError),
+                                availableVersion.ToDisplayString(),
+                                new CSharpRequiredLanguageVersion(requiredVersion));
```

Comment:
Consider making a local (outside the loop) for the CSharpRequiredLanguageVersion.

Also, please take a note to add this error code (ERR_DefaultInterfaceImplementationModifier) to the list monitored by the UpgradeProject code fixer. 
This PR can be used to find the relevant code and test files: [https://github.com/dotnet/roslyn/pull/18045](https://github.com/dotnet/roslyn/pull/18045)